ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -156,7 +156,31 @@ def forward(
         return encoding, memories
 
 
-class ValueNetwork(nn.Module):
+class Critic(abc.ABC):
```

Comment:
I think it is time we split networks.py into networks.py, actor.py and Critic.py. Maybe in a subsequent PR.